### ISSUE TITLE
Use specific extensions

### ISSUE THREAD

**Issue by mjmaix** [leifoolsen/mdl-ext#84]
Thank you for the great extension pack.

I'd like to use this but one concern is that the minified js + css is 194kb, although it's small, it is still a problem if I'll only use the extended menu button.

Is there a way to only import the minimum requirements for a component?

**Comment by leifoolsen** [comment]
Hello @mjmaix. Thank you for reporting this. 

When creating the JavaScript components I had to follow the MDL programming model. Due to the fact that the MDL components are placed in global scope, I don't think it's possible to cherry pick components using import or require. I will do some tests and see if I can use Webpack to split MDL-EXT components into separate files. 

The SASS modules are relatively simple to split, just @import the SASS modules required for a component. 

I have created an example which only imports SASS needed for the menu button here: https://github.com/leifoolsen/my-playground/tree/master/mdlext-stripped

I'll use the same project for experimenting with cherry picking of JS components.
